modules/openapi-generator/src/main/resources/php/ObjectSerializer.mustache

```diff
@@ -58,7 +58,7 @@ class ObjectSerializer
                 if ($value !== null
                     && !in_array($openAPIType, [{{&primitives}}], true)
                     && method_exists($openAPIType, 'getAllowableEnumValues')
-                    && !in_array($value, $openAPIType::getAllowableEnumValues())) {
+                    && !in_array($value, $openAPIType::getAllowableEnumValues(), true)) {
```

Comment:
Can you explain, please, how this change is related to `phpcs` fixer?

Reply:
`php-cs-fixer` raises a `strict-param` warning for in_array calls where the `$strict` parameter is false (which is the default). Passing true ensures we use strict comparison, i.e. value + type checking. 

From my reading of the code I couldn't see any reason why we would *need* loose comparisons, especially since they can result in weird behaviour as documented [here](http://php.net/manual/en/function.in-array.php#106319) and [here](http://php.net/manual/en/function.in-array.php#121479).